framework/src/org/checkerframework/common/value/ValueAnnotatedTypeFactory.java

```diff
@@ -133,6 +138,8 @@ public ValueAnnotatedTypeFactory(BaseTypeChecker checker) {
         // this line just registers the alias. The BottomVal is never used.
         addAliasedAnnotation(MinLen.class, BOTTOMVAL);
 
+        addAliasedAnnotation(PolyAll.class, POLY);
```

Comment:
PolyAll should be treated as PolyValue by the framework.  So this shouldn't be needed.